setup.cfg

```diff
@@ -16,7 +16,7 @@ url = https://github.com/opendatacube/odc-tools/
 include_package_data = true
 zip_safe = false
 packages = find_namespace:
-python_requires = >=3.6
+python_requires = >=3.8
```

Comment:
Note that Python 3.8 is no longer getting security updates and the most recent versions of datacube-core require python 3.9+.